dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,10 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        import dstep.core.Traits : isKeyword;
+        if (ident.spelling.isKeyword())
+            return false;
```

Comment:
Looking at this problem in general, I don't see why adding a `_` suffix wouldn't work. Example:

```c
void delegate(bool condition);
#define PROTOBUF_C_DELEGATE(condition) delegate(condition)
```

I don't see why that couldn't be translated to:

```d
pragma(mangle, "delegate") void delegate_(bool condition);
alias PROTOBUF_C_DELEGATE = delegate_;
```

Your assumption in this PR is also that `assert` will always be the symbol defined in `assert.h`, which might not be the case. At least how the code is written now, it will ignore all symbols that are keywords in D, which might not be desirable.

In this particular case with `assert`, I don't think anyone would think it's useful to translate that function/macro to D (a corresponding `assert` module does not exist in druntime). If someone would manually translate some C code using `assert` they would most likely translate the use of `assert` to the D built-in `assert` and not translate the actual macro/function.

I'm not sure what's the best short term solution is. One idea would be to special case `assert` and implement the `assert` macro inline, or completely ignore it.

For a long term solution, I've been thinking for a while about a generic way to specify declarations that should be ignored.